### ISSUE TITLE
Map bad/missing value-type 

### ISSUE THREAD

**Issue by jskov-jyskebank-dk** [jskov/openapi-jaxrs-client#596]
```yaml
    FlagDto:
      type: object
      properties:
        property:
          type: string
          nullable: true
        name:
          type: string
          nullable: true
        description:
          type: string
          nullable: true
        flagSeverity:
          $ref: '#/components/schemas/FlagSeverityEnum'
        isTriggered:
          type: boolean
        data:
          type: object
          additionalProperties: {}
          nullable: true
      additionalProperties: false
    FlagSeverityEnum:
      enum:
        - disabled
        - info
        - riskIndicator
        - warning
      type: string
```
results in this code:

```java
@JsonSerialize(include = com.fasterxml.jackson.databind.annotation.JsonSerialize.Inclusion.NON_NULL)
@javax.annotation.processing.Generated(value = "dk.mada.jaxrs.Generator")
public class FlagDto implements Serializable {
  public static final String JSON_PROPERTY_DATA = "data";
  @JsonProperty(JSON_PROPERTY_DATA)
  @Schema(nullable = true)
  private Map<String, <VALIDATION>> data;

  public static final String JSON_PROPERTY_DESCRIPTION = "description";
  @JsonProperty(JSON_PROPERTY_DESCRIPTION)
  @Schema(nullable = true)
  private String description;

```

So the map data type is not processed correctly.

(claus)

**Comment by jskov-jyskebank-dk** [comment]
```yaml
content:
    application/json:
      schema:
        type: object
        additionalProperties:
          type: string
```

Workaround is to specify a type for the properties

So generator should probably have an option to control a default fall-back type (Object, prolly)

**Comment by jskov** [comment]
fixed by #611
